.circleci/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 2.1
+orbs:
+  node: circleci/node@1.1.6
+jobs:
+  build:
+    executor:
+      name: node/default
+    steps:
+      - checkout
+      - node/with-cache:
+          steps:
+            - run: yarn install
+workflows:
+    build:
+      jobs:
+        - build
```

Comment:
Are you planning on setting up FP as part of a separate PR?

Reply:
Yes to FP in a separate PR :)